### PR TITLE
[ssr] make `>` stand alone

### PR DESCRIPTION
**Pull request by gares**
Depends: #6705

### CODE REVIEW

**Comment by gares** [comment]
CC @charguer

File: doc/sphinx/proof-engine/ssreflect-proof-language.rst [hunk 1]
```diff
@@ -1588,21 +1594,43 @@ When an :token:`s_item` bears a :token:`clear_switch`, then the
...
+|SSR| supports the following :token:`i_pattern`\s:
 
 :token:`ident`
   pops the top variable, assumption, or local definition into
   a new constant, fact, or defined constant :token:`ident`, respectively.
   Note that defined constants cannot be introduced when δ-expansion is
   required to expose the top variable or assumption.
+``>``
+  pops every variable occurring in the rest of the stack.
+  Type class instances are popped even if then don't occur
```
Comment (Zimmi48) [inline doc/sphinx/proof-engine/ssreflect-proof-language.rst]:
```suggestion
  Type class instances are popped even if they don't occur
```

**Review by Zimmi48 — APPROVED** [review]

**Comment by gares** [comment]
@maximedenes this is ready now since it's dependency was merged.

**Review by CohenCyril — APPROVED** [review]
Looks good to me

**Review by maximedenes — APPROVED** [review]
Looks very good! The code is in fact now simpler, and the feature more powerful.

**Pull request merged**